add note to docs to explain pre-hook and post-hook examples (#1054)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/pre-hook-post-hook.md

<TabItem value="models">

> In the following examples,  we use the `|` symbol to separate two different formatting options for SQL statements in pre-hooks and post-hooks. The first option (without brackets) will accept a single SQL statement as a string, while the second (with brackets) will accept multiple SQL statements as an array of strings.

Reviewer: We should make these snippets and then ship this pr.

> **Note**: In the following examples, the `|` symbol is meant to separate two different formatting options for SQL statements in pre-hooks and post-hooks. The first option (without brackets) will accept a single SQL statement as a string, while the second (with brackets) will accept multiple SQL statements as an array of strings.

Reviewer: I love this suggestion! I think it's more important than a note and would move it to the top of the page as a paragraph. Then remove the other two notes. My text suggestion just makes the sentence active.

```suggestion
In the following examples,  we use the `|` symbol to separate two different formatting options for SQL statements in pre-hooks and post-hooks. The first option (without brackets) will accept a single SQL statement as a string, while the second (with brackets) will accept multiple SQL statements as an array of strings.
```

<File name='dbt_project.yml'>

> ```yml

Reviewer: (The following comment is mostly to give you more context.)
Another thing I've been slowly trying to standardize on is using all caps for [dynamic text](https://github.com/dbt-labs/product-docs-team/blob/main/content-style-guide.md#dynamic-text) (text that users need to replace with their own words), for example, `SQL_STATEMENT` instead of `<sql-statement>`  because people get confused with whether they should  `<>` or not. 

I created [an issue ](https://github.com/dbt-labs/docs.getdbt.com/issues/1056)to sweep through the docs and make this change. 

Once I do ☝🏻, I will add something in the same paragraph that tells people to replace SQL_STATEMENT with whatever they should be providing. Maybe the name of the SQL statement they are using?
